### PR TITLE
added packages tests support 

### LINKED ISSUE #90: Add support for packages tests
See doc.

### PR DESCRIPTION
**Pull request by i582**
Fixes #90

### CODE REVIEW

File: src/main/kotlin/com/vk/admstorm/utils/MyPathUtils.kt [hunk 1]
```diff
@@ -50,6 +50,9 @@ object MyPathUtils {
     }
 
     fun relativeLocalPath(project: Project, path: String): String {
+        if (path.isEmpty()) {
+            return ""
+        }
         val projectDir = resolveProjectDir(project) ?: return path.normalizeSlashes()
```
Comment (Danil42Russia) [inline src/main/kotlin/com/vk/admstorm/utils/MyPathUtils.kt:56]: missing a new line

File: src/main/kotlin/com/vk/admstorm/utils/MyUtils.kt [hunk 2]
```diff
@@ -199,4 +205,14 @@ object MyUtils {
             }
         }, delay)
     }
+
+    fun String.unquote(): String {
+        if (length >= 2 &&
```
Comment (Danil42Russia) [inline src/main/kotlin/com/vk/admstorm/utils/MyUtils.kt:210]: replace with `return removeSurrounding("\"").removeSurrounding("'")`

**Review by Danil42Russia — CHANGES_REQUESTED** [review]

File: src/main/kotlin/com/vk/admstorm/utils/MyPathUtils.kt (hunk 1 above)
Comment (i582) [reply src/main/kotlin/com/vk/admstorm/utils/MyPathUtils.kt:57]: Fixed.

File: src/main/kotlin/com/vk/admstorm/utils/MyUtils.kt (hunk 2 above)
Comment (i582) [reply src/main/kotlin/com/vk/admstorm/utils/MyUtils.kt]: Fixed.

**Pull request merged**